baseapi.py

```diff
@@ -115,4 +130,4 @@ def request(self, url, method, params=None, data=None):
         except ValueError:
             raise JSONDecodeError()   
 
-        return json['d']
+        return json
```

Comment:
What keys does ``json`` contain? What was this ``['d']`` key that was returned before?